docs/SPEC.md

```diff
@@ -1723,20 +1723,40 @@ from(bucket: "telegraf/autogen")
 	|> median()
 ```
 
-##### Moving Average
+#####  Moving Average
 
 Moving Average is an aggregate operation.
-For each aggregated column, it means the values of the following records for a defined window range.
+For each aggregated column, it means the values of the following records for a defined number of points.
```

Comment:
Can you change this to 

```
Moving Average is an aggregate operation.
It means the values of the following records for a defined number of points, for the specified aggregated column.
```

There is  only one column on which it operates, right?